src/DocScan/Service.php

```diff
@@ -28,6 +28,9 @@
 
 class Service
 {
+    /** @const int */
+    private const HTTP_STATUS_NO_CONTENT = 204;
```

Comment:
Do we have other similar constants defined elsewhere? just wondering if this should live somewhere else

Reply:
For now - not)